packages/flutter/lib/src/services/asset_bundle.dart

```diff
@@ -82,7 +82,7 @@ abstract class AssetBundle {
   Future<String> loadString(String key, { bool cache = true }) async {
     final ByteData data = await load(key);
     if (data == null) {
```

Comment:
This should be removed. But the analyzer doesn't report with an unnecessary null check.